scroll-animations-1/Overview.bs

```diff
@@ -348,11 +354,14 @@ behavior under different directionalities and writing modes.
 <pre class="idl">
 enum ScrollTimelineAutoKeyword { "auto" };
 
+typedef (CSSNumberish or ScrollTimelineAutoKeyword) ContainerBasedOffset;
```

Comment:
We should IMO also accept `CSSKeywordValue` matching `auto`.

Would you want to use `CSSKeywordish` here (if it existed)?

Reply:
My bad, CSSKeywordish already exists: https://drafts.css-houdini.org/css-typed-om-1/#typedefdef-csskeywordish

Do we want to use that somehow, then?

Reply:
Done. As part of this I now have a procedure that rectifies strings into CSSKeyword and rejects invalid values at the time of setting the value.